docs/content/testing/writing-a-new-test.md

```diff
@@ -0,0 +1,344 @@
+**Warning:** The [pre-alpha](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) release is for evaluation purposes only, with limited documentation - see the guidance on [Recommended use](../../../README.md#recommended-use).
+
+-----
+
+# Writing a new test
+
+This document walks through creating tests for a made-up class to explain how
+ the parts of the Unity GDK can be tested.
+
+## Prerequisites
+
+Please read through and understand the [Testing Overview](./testing-overview.md)
+ and the [Testing Guidelines](./testing-guidelines.md) documents first.
+
+## Using NUnit
+
+To help get started with testing, we recommend these documents which help cover
+ the basics:
+
+- https://www.youtube.com/watch?v=1TPZetHaZ-A 
```

Comment:
would use a different link title:
`[Introduction To NUnit](https://www.youtube.com/watch?v=1TPZetHaZ-A )`